### PR TITLE
Add faq for why parameter on experiment is good

### PR DESCRIPTION
**Pull request by jkw-statsig**

### CODE REVIEW

**Comment by jasonwzm** [comment]
🚀 Deployed on https://62da04ce60d1c62972938550--cozy-fox-0defec.netlify.app

File: docs/faq.mdx [hunk 1]
```diff
@@ -14,9 +14,32 @@ with more SDKs, plugins, and integrations shipping all the time.
...
+    color = 'BLACK';
+} else if (otherExpEngine.getExperiment('button_color_test').getGroup() === 'Blue') {
+    color = 'BLUE';
+}
+```
+
+In Statsig, you will add a parameter to your experiment named "button_color", and your code will look like this:
+
+```
+color = statsig.getExperiment('button_color_test').getString('button_color', 'BLACK');
```
Comment (tore-statsig) [inline docs/faq.mdx:40]: The experiment name is still hard coded here, so I feel the layers example is better

**Review by tore-statsig — APPROVED** [review]
Also documented here:

https://docs.statsig.com/experiments-plus/getting-group

But sure

File: docs/faq.mdx (hunk 1 above)
Comment (jkw-statsig) [reply docs/faq.mdx:40]: Layer is better but this is to get people to understand even without layer, why parameter is still better than group. The idea here is that you can restart the experiment with different values and more groups without code change

**Pull request merged**